lib/sisimai/smtp/error.rb

```diff
@@ -85,17 +79,17 @@ def soft_or_hard(argv1 = '', argv2 = '')
             softorhard = ''
 
           elsif argv1 == 'onhold' || argv1 == 'undefined'
-            # Check with the value of D.S.N. in $argv2
+            # Check with the value of D.S.N. in argv2
             getchecked = Sisimai::SMTP::Error.is_permanent(argv2)
 
             if getchecked.nil?
               softorhard = ''
             else
               softorhard = if getchecked
-                'hard'
-              else
-                'soft'
-              end
+                             'hard'
```

Comment:
__[rubocop]__
* Layout/IndentationWidth: Use 2 (not 15) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)

<!-- Comment by [SideCI](https://sideci.com) -->